integrated_channels/sap_success_factors/transmitters/courses.py

```diff
@@ -14,28 +16,37 @@
 class SuccessFactorsCourseTransmitter(SuccessFactorsTransmitterBase):
     """
     This endpoint is intended to carry out an export of course data to SuccessFactors for a given Enterprise.
-
-    Implementation will look something like:
-
-    Perform basic validation on the input (e.g. is this Enterprise configured for SuccessFactors?)
-
-    Retrieve oauth access token from SuccessFactors, based on OCNWebServicesEnterpriseCustomerConfiguration.
-    This can either initiate generating a new token, or using an unexpired token which we have cached.
-
-    Find which catalog is associated with the enterprise customer.
-
-    Retrieve course data in the catalog from the Catalog API.
-
-    Transform course data into the format expected by SuccessFactors.
-
-    Send the transformed course data to SuccessFactors using the configured endpoint
-    (taken from OCNWebServicesConfiguration and OCNWebServicesEnterpriseCustomerConfiguration) and the oauth token.
-
-    Record information about the success/failure of posting the course data to SuccessFactors in
-     CourseContentExportAudit.
     """
-    def transmit(self, courseware_exporter):
+
+    def transmit(self, payload):
         """
-        Get serialized data from the courseware exporter and transmit it.
+        Send a course data import call to SAP SuccessFactors using the client.
+
+        Args:
+            payload (SapCourseExporter): The OCN course exporter object to send to SAP SuccessFactors
         """
-        LOGGER.info(courseware_exporter.get_serialized_data())
+        serialized_payload = payload.get_serialized_data()
+        LOGGER.info(serialized_payload)
```

Comment:
After this lands in production, do we still want it to be treated as INFO, or should it be changed to DEBUG?